### ISSUE TITLE
How to do checkpoint and use the ‘’resume‘’ option for task.fit (image classification)

### ISSUE THREAD

**Issue by dsp6414** [awslabs/autogluon#424]
can anyone help me?

**Comment by dsp6414** [comment]
i can save and load the model ,and also can export the model  mxnet json format and model params, but i still don't know how  to do checkpoint in the next task.fit when i change the args.

print(classifier)
print(classifier.model)
classifier.save('classification_test')


classifier2=Classifier.load("classification_test")
test_acc = classifier2.evaluate(test_dataset)
print('Top-1 test acc: %.3f' % test_acc)


import gluoncv as gcv
from gluoncv.utils import export_block


export_block('shopee_iet',classifier.model , preprocess=True, layout='HWC')

print('Done.')

**Comment by zhreshold** [comment]
I don't understand, can you elaborate what you are trying to do?

**Comment by dsp6414** [comment]
Sorry, I'd like to know how to use the last checkpoint for fine-tuning in new training when I changed the new hyper-params but the model leave the same.

**Comment by mseeger** [comment]
Could you be more specific? Are you trying to checkpoint the scheduler state, and then resume an experiment from a checkpoint, using the resume argument of the scheduler?

If so, this feature is not yet fully supported across all schedulers and searchers. Please note it does not checkpoint the models you evaluate, but only the optimizer state. So there should be no need to save MXNet models, unless the searcher itself relies on such models.
